crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1147,7 +1147,6 @@ def upgrade_one_node(name)
           )
         else
           node_api.upgrade
-          node_api.reboot_and_wait
```

Comment:
what happened to this?

Reply:
The point is that `node_api.upgrade` already includes `reboot_and_wait` call.
Nothing that bad, it should not reboot twice (that's why we have that check for `rebooted_file`), but anyway it's not necessary.

Reply:
OK. Would be good to mention it in the PR description. I haven't noticed the second commit.